File: 20190519python/0027. Remove Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        left=0
        right=len(nums)-1
        while(left<=right):
            if nums[left]==val:
                nums[left]=nums[right]
                right-=1
            else:
                left+=1
        return left

File: archives/C#/0027. Remove Element.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {        
        int cur=0;
        for (int i=0;i<nums.Length;i++){
            if (nums[i]!=val){
                nums[cur]=nums[i];
                cur++;
            }
        }
        return cur;
    }
}
